caravel/assets/visualizations/big_number.css

```diff
@@ -1,4 +1,4 @@
-.big_number g.axis text {
+.big_number g.axis text, .big_number_total g.axis text {
```

Comment:
nit, but style-wise (https://github.com/airbnb/css#formatting) each selector should have its own line when using multiple selectors in a rule declaration.